src/Integration/Connection/ConnectionWorkflow.cs

```diff
@@ -271,6 +272,13 @@ private static bool IsTestProjectPatternProperty(SonarQubeProperty property)
         private async Task<bool> AreSolutionProjectsAndSonarQubePluginsCompatibleAsync(IProgressController controller,
             IProgressStepExecutionEvents notifications, CancellationToken cancellationToken)
         {
+            var folderWorkspaceService = host.GetMefService<IFolderWorkspaceService>();
+
+            if (folderWorkspaceService.IsFolderWorkspace())
+            {
+                return true;
```

Comment:
This isn't strictly correct. If I remember correctly, the purpose of this method was supposed to be to only allow binding if there are bindable projects with the corresponding plugins that could analyse those project types e.g. if you had a C++ project but don't have the C++ plugin because you are using Community edition, you couldn't bind.

Here, we don't know which languages are being used in the solution, so we shouldn't just return true.

However, it's a corner case, so perhaps just create a ticket for it to do "later".

Reply:
You're right, the logic here should also check if there are bindable languages. I'll add a ticket.